variants/metal-dev/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "metal-dev"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant.image-layout]
+partition-plan = "unified"
+data-image-size-gib = 5
+
+[package.metadata.build-variant]
+image-format = "raw"
+supported-arches = ["x86_64"]
+kernel-parameters = [
+    "console=ttyS1,115200n8",
```

Comment:
Considering that this will be different for most hardware configurations, we will need to come up with a more "dynamic" way to pass these parameters to the kernel command line and runtime.  For now since this image must be built by the developer, we will leave this up to them to change at build time.

Reply:
I don't think we can change the kernel command line at runtime if we support secure boot. I might be wrong :thinking: .